ansible/templates/whisk.properties.j2

```diff
@@ -86,3 +86,8 @@ db.password={{ db_password }}
 db.prefix={{ db_prefix }}
 db.whisk.actions={{ db_prefix }}whisks
 db.whisk.auths={{ db_auth }}
+db.whisk.apigw={{ db_apigw }}
+
+apigw.auth.user={{apigw_auth_user}}
+apigw.auth.pwd={{apigw_auth_pwd}}
+apigw.host={{apigw_host}}
```

Comment:
why not change this to `http://{{ groups['apigateway']|first }}:{{apigateway.port.api}}/v1`
and remove from environment group vars? much cleaner this way and less overhead.